code/modules/admin/admin.dm

```diff
@@ -611,6 +611,24 @@ GLOBAL_VAR_INIT(nologevent, 0)
 	feedback_add_details("admin_verb","R") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 	world.Reboot("Initiated by [usr.client.holder.fakekey ? "Admin" : usr.key].", "end_error", "admin reboot - by [usr.key] [usr.client.holder.fakekey ? "(stealth)" : ""]", delay)
 
+/datum/admins/proc/smoothend()
+	set category = "Server"
+	set name = "Annouce game over"
+	set desc = "Instantly ends round and brings up objectives like shadowlings or wizards dying."
+
+	if(!check_rights(R_SERVER))
+		return
+
+	if(alert(usr, "Are you sure you want to admin end the round and bring up objectives?", "End the game", "Yes", "No") != "Yes")
+		return
+
+	message_admins("[key_name_admin(usr)] has admin ended the round.")
+	log_admin("[key_name(usr)] has admin ended the round")
+	SSticker.force_ending = 1
+	to_chat(world, "<span class='warning'><FONT size = 3><b>The administrators have declared the round over and have ended the round!</b></FONT></span>")
+	feedback_add_details("admin_verb","S") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+	feedback_set_details("round_end_result","Admin ended")
+
```

Comment:
```suggestion
/datum/admins/proc/smoothend()
	set category = "Server"
	set name = "End Round"
	set desc = "Instantly ends the round and brings up the scoreboard, like shadowlings or wizards dying."

	if(!check_rights(R_SERVER))
		return

	var/input = sanitize(copytext(input(usr, "What text should players see announcing the round end?", "Specify Announcement Text", "Shift Has Ended!"), 1, MAX_MESSAGE_LEN))	
	
	if(!input)
		return

	if(SSticker.force_ending)
		return

	message_admins("[key_name_admin(usr)] has admin ended the round with message: '[input]'")
	log_admin("[key_name(usr)] has admin ended the round  with message: '[input]'")
	SSticker.force_ending = 1
	to_chat(world, "<span class='warning'><FONT size = 3><b>[input]</b></FONT></span>")
	feedback_add_details("admin_verb", "S") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
	feedback_set_details("round_end_result", "Admin ended")

```
Reasoning:
1) Should be called "End Round".
2) Should prompt for a reason and display that reason to players, so we can declare e.g. "(event antag) Major Victory" as the reason.
3) Should log the reason as well.
4) Cancelling should be done by clicking "cancel" or not entering a reason on the text input box.
5) Proc calls need proper spacing for their arguments.
6) Sensible to have a check after the input to prevent accidental double-use.

Other feedback:
- Why "S" for the feedback_add_details, instead of say "END" or "ER"? Are those in use?